src/hooks/useNowDate.ts

```diff
@@ -1,11 +1,12 @@
-import { ref } from "vue";
+import { ref, Ref } from "vue";
 import { useInterval } from "./useInterval";
 
-export const useNowDate = () => {
+export const useNowDate = (slideDate: Ref<number>) => {
   const now = ref(new Date());
 
   useInterval(() => {
     now.value = new Date();
+    now.value.setHours(slideDate.value);
```

Comment:
It will always change hours to offset. If offset is 12, then hours will be sent to 12.

Let's add offset to current time.
```suggestion
    const currentH = now.value.getHours()
    now.value.setHours(currentH + slideDate.value);
```